src/Services/SmartProxy.js

```diff
@@ -12,8 +12,13 @@ export const smartProxyApi = createApi({
       query: (id, includeDisabled = true) =>
         `clusters/${id}/report?get_disabled=${includeDisabled}`,
     }),
-    // Get rule's content using id in the rule_plugin_name|error_key format
-    getRuleById: builder.query({ query: (id) => `rules/${id}/content` }),
+    // Get rule's content using id (recId = recommendation id) in the rule_plugin_name|error_key format
+    getRuleById: builder.query({
+      query: (recId) => `rules/${recId}/content`,
+    }),
+    getAffectedClusters: builder.query({
+      query: (recId) => `rule/${recId}/clusters_detail`,
```

Comment:
looks like yah have a data.data wrapper for this response (from the component code above) consider ah transform to unwrap that a lil bit

`transformResponse: (response) => response.data`

Reply:
o yah here https://github.com/RedHatInsights/ocp-advisor-frontend/pull/32/files#diff-a99352fe80e06fe6ebaefa3bc326528f0d4d2f0887e9db000de6d61e221b272aR40